Sources/TuistKit/Generator/Generator.swift

```diff
@@ -80,7 +80,11 @@ public class Generator: Generating {
     }
 
     func load(path: AbsolutePath) async throws -> (Graph, [SideEffectDescriptor]) {
+        logger.notice("Loading and constructing the graph", metadata: .section)
+        logger.notice("It might take a while if the cache is empty")
```

Comment:
I'm not sure if a user understand what we're actually caching. I get the idea of this, but I'd probably in favor of removing this specific log.

Reply:
The manifests. The first time you run it with the manifests cache empty, there's no output, and it gives the impression that the CLI got stack. I can change the wording a bit.

Reply:
I do understand what we're caching. I'm saying the user might not and then this log would just be confusing.